README.md

```diff
@@ -1,9 +1,5 @@
 # metal-python
 
-
-[![Experimental](https://img.shields.io/badge/Stability-Experimental-red.svg)](https://github.com/equinix-labs/equinix-labs/blob/main/uniform-standards.md)
-
-> **[Experimental](https://github.com/equinix-labs/equinix-labs/blob/main/experimental-statement.md)**
 > This is experimental. Don't use it in production. Examples demonstrate that this client is usable. Please submit patches and open issues with your experience. This repo contains Python code generated from a customized [metal_openapi.fixed.yaml](metal_openapi.fixed.yaml) based on the [Equinix Metal API spec](https://api.equinix.com/metal/v1/api-docs). The client is generated using the python-nextgen generator built into the [OpenAPITools openapi-generator](https://github.com/OpenAPITools/openapi-generator).
```

Comment:
This warning was added to the project ahead of the 0.1.0 release in March 2023. We've since solidified the maintenance and usage patterns through CI actions and example projects. This project is in use by the Ansible collection which does not provide the same warning. 

https://github.com/equinix-labs/ansible-collection-equinix?tab=readme-ov-file#equinix-ansible-collection

While we are removing the Experimental labels, we can remove the disclaimer and advise on this line.  We are removing the labels because the projects are offered as-is, per the license files. Use at your own risk is part of that license.

Removal of the Experimental warnings does not explicitly endorse the project's production readiness or imply a long-term commitment to it. In fact, we have been considering an equinix-sdk-python project to consolidate Metal, Fabric, and other Equinix Python SDK needs.

```suggestion
> Examples demonstrate that this client is usable. Please submit patches and open issues with your experience. This repo contains Python code generated from a customized [metal_openapi.fixed.yaml](metal_openapi.fixed.yaml) based on the [Equinix Metal API spec](https://api.equinix.com/metal/v1/api-docs). The client is generated using the python-nextgen generator built into the [OpenAPITools openapi-generator](https://github.com/OpenAPITools/openapi-generator).
```